### DISCUSSION TITLE
User created in USERS table but not auth_identities

### DISCUSSION THREAD

**Discussion by mauriceb77** [codeigniter4/shield#768] [Q&A]
I recently modified my localhost to use a central/shared installation of CI4 so I could work on multiple sites locally. Before this I only had a single site on my localhost with Shield and all worked well. I even deployed my site to a dedicated webserver and all is well.

After the modification to the CI4 installation, I am now not able to login with existing accounts or register new accounts. I receive error "Trying to access array offset on value of type null" during registration. I can see a new user added to the user table with a new ID but the auth_identities table is not inserted.

**Comment by kenjis** [comment]
> After the modification to the CI4 installation, I am now not able to login with existing accounts or register new accounts.

You did something wrong with the modification. 
But nobody knows what it is, because we cannot see your screen or source code.
